libraries/AP_VideoTX/AP_SmartAudio.cpp

```diff
@@ -141,13 +138,15 @@ void AP_SmartAudio::loop()
             if (AP::vtx().have_params_changed() ||_vtx_power_change_pending
                 || _vtx_freq_change_pending || _vtx_options_change_pending) {
                 update_vtx_params();
-                _vtx_gcs_pending = true;
+                _vtx_changes_pending = true;
+                vtx.set_configuration_done(false);
```

Comment:
You should have two mutators here - "set_configuration_finished() and set_configuration_pending()"

Reply:
Sure